readthedocs/api/v3/proxied_views.py

```diff
@@ -1,13 +1,11 @@
 from readthedocs.core.utils.extend import SettingsOverrideObject
 from readthedocs.embed.v3.views import EmbedAPIBase
+from readthedocs.core.mixins import AuthenticatedClassesMixin
 
 
-class ProxiedEmbedAPIBase(EmbedAPIBase):
-
-    # DRF has BasicAuthentication and SessionAuthentication as default classes.
-    # We don't support neither in the community site.
-    authentication_classes = []
-
+class ProxiedEmbedAPIBase(EmbedAPIBase, AuthenticatedClassesMixin):
```

Comment:
The mixin class should be listed first, otherwise it won't override things from `EmbedAPIBase`.

Reply:
got it, changing this

Reply:
@stsewd I've fixed this as well